downstream/modules/platform/ref-ports-firewalls-requirements.adoc

```diff
@@ -35,54 +37,61 @@ ALLOW connections from controller(s) to database port
 .{ControllerNameStart}
 [options="header"]
 |===
-|Port |Protocol |Service |Installer Inventory Variable |Required for
+|Port |Protocol |Service |Direction |Installer Inventory Variable |Required for
 |22
 |TCP
 |SSH
-|ansible_port
+|Bi-directional
+|`ansible_port`
 |Installation
 
 |80
 |TCP
 |HTTP
-|nginx_http_port
+|Incoming
```

Comment:
```suggestion
|Inbound
```

Reply:
As a global comment I believe "inbound" and "outbound" are the conventional terms for network traffic directions. I think I got all the instances but would suggest maybe changing any others.

I also wonder about "bi-directional". Personally I would probably favour saying "Inbound and outbound" instead, even if it is more words. That makes it explicit and might help with international audiences. But I'll leave that to you.

Reply:
@oraNod Thanks for the suggestion. I will make those changes.